packages/cli/lib/index.js

```diff
@@ -82,26 +79,31 @@ prog
 	.describe('Start a live-reload server for development')
 	.option('--src', 'Specify source directory', 'src')
 	.option('--cwd', 'A directory to use instead of $PWD', '.')
-	.option('--esm', 'Builds ES-2015 bundles for your code.', false)
+	.option('--esm', 'Builds ES-2015 bundles for your code', false)
 	.option('--clear', 'Clear the console', true)
 	.option('--sw', 'Generate and attach a Service Worker', undefined)
-	.option('--babelConfig', 'Specify the babel config file', '.babelrc')
+	.option('--babelConfig', 'Path to custom Babel config', '.babelrc')
```

Comment:
Changed the verbiage here purely for consistency. 

"Path to PEM key for custom SSL..."
"Path to custom SSL..."
"Path to custom HTML..."
"Path to custom CLI..."

"Path to custom Babel config" seems to fit better.